en/CommandLine.md

```diff
@@ -117,6 +118,17 @@ java -jar JabRef.jar -o path\split,xmp  path\entries.bib -n
 ```
 
 
+### Import BibTeX: `-importBibtex`
+
+Import or load code directly from the BibTeX file.
```

Comment:
Is it possible to make a whole paragraph out of it? Not just one sentence, one paragraph? Could you try? Maybe, just removing the empty lines helps. Maybe, you come up with some glue words?

Reply:
@koppor I have expanded upon the description for the `-importBibtex` command line option as per your request. Please let me know if additional changes need to be made.